expressions/methods.md

```diff
@@ -196,6 +196,21 @@ class Foo
 
 __Can I call using positional arguments but miss out the first one?__ No. If you use positional arguments they must be the first ones in the call.
 
+## Chaining
+
+Method chaining allows you to chain calls on an object without requiring the method to return its receiver. The syntax to call a method and chain the receiver is `object.>method()`, which is roughly equivalent to `(object.method() ; object)`. Chaining a method discards its original return value.
```

Comment:
I think "original return value" might be hmmm... not sure of the word.

Perhaps "normal return value" ?